liquibase-core/src/main/java/liquibase/snapshot/jvm/ColumnAutoIncrementService.java

```diff
@@ -116,7 +117,7 @@ private String getQueryForDatabase(Database database) throws LiquibaseException
                         "    JOIN pg_depend d ON c.oid = d.objid " +
                         "    JOIN pg_class td ON td.oid = d.refobjid " +
                         "    JOIN pg_attribute pa ON  pa.attrelid=td.oid AND pa.attnum=d.refobjsubid " +
-                        "WHERE c.relkind = 'S' AND d.deptype = 'a'";
+                        "WHERE c.relkind = 'S' AND d.deptype = 'a' AND ns.nspname = '" + schema.getName() + "'";
```

Comment:
There are a couple other `if` branches we should make sure are respecting the new schema name